.github/workflows/docs.yml

```diff
@@ -28,12 +28,12 @@ jobs:
           poetry-version: "1.2.2"
 
       - run: poetry install --all-extras
-      - run: mkdir -p docs
-      - run: poetry run pdoc --docformat=numpy -o docs landingai
+      - run: mkdir -p docs-build
+      - run: poetry run pdoc -t docs/templates --docformat=numpy -o docs-build landingai
```

Comment:
Separating the output in a different directory, to avoid mixing the generated assets with the source.